tinygrad/tensor.py

```diff
@@ -472,7 +472,19 @@ def transpose(self, ax1=1, ax2=0) -> Tensor:
     order[ax1], order[ax2] = order[ax2], order[ax1]
     return self.permute(order)
   def flatten(self, start_dim=0): return self.reshape(shape=self.shape[:start_dim] + (-1,))
-
+  def cross(self, other, dim=0):
```

Comment:
I think we move this function next to the other Binary ops no? From what I understand cross would operate on two tensors and run elementwise to return one.

Reply:
I agree, I moved it to binary mlops and reduced line count